pkg/command/command.go

```diff
@@ -30,6 +30,7 @@ type runner interface {
 	AppendEnv(env []string)
 	Stdout(out io.Writer)
 	Stderr(err io.Writer)
+	GetExitCode() int
```

Comment:
There isn't a reason to add this here. "runner" is not of the interface type, but of `command.Command{}` where the method is used.

Reply:
You are right. It was needed when used inside `runSonar` where `runner` is a `command.ExecRunner`.

Reply:
removed